reference/tools/system/package_manager.rst

```diff
@@ -31,7 +31,13 @@ You can use these tools inside the ``system_requirements()`` method of your reci
 Conan will automatically choose which package manager to use by looking at the Operating
 System name. In the example above, if we are running on Ubuntu Linux, Conan will ignore
 all the calls except for the ``Apt()`` one and will only try to install the packages using the
-``apt-get`` tool. Conan uses the following mapping by default:
+``apt-get`` tool.
+
+The ``package_manager`` tool allows you to specify the package version of the system package
+to be installed using the format ``<package-name>=<package-verison>``, for example ``Apt(self).install(["libgl-dev=0.0.1"])``.
+On systems where package versioning is not supported by the package manager, the provided version will be ignored.
```

Comment:
I think mentioning those package managers that don't support versions would be nice